### PR TITLE
Fixes #25490 - fix html rendering in tooltips

### PR DESCRIPTION
**Pull request by amirfefer**
screenshot from 2018-11-19 17-58-00

### CODE REVIEW

**Review by ekohl — DISMISSED** [review]
Was this rendered in Chrome? Was that a (very unlikely) potential XSS there?

**Review by waldenraines — DISMISSED** [review]
LGTM

**Comment by amirfefer** [comment]
@ekohl - yes, it rendered in chrome as well. and I escaped few variables that have some sort of xss potential

File: app/assets/javascripts/charts.js [hunk 1]
```diff
@@ -240,7 +240,7 @@ function reset_zoom(item){
 
 function showTooltip(pos, item, formater) {
   var content = formater(item);
-  $('<div id="flot-tooltip">').text(content).css({
+  $('<div id="flot-tooltip">').html(content).css({
```
Comment (tbrisker) [inline app/assets/javascripts/charts.js]: Would it maybe be better to remove the `<b>` tags and keep this as text? that way we don't open the possibility of xss here.

**Comment by ekohl** [comment]
> @ekohl - yes, it rendered in chrome as well. and I escaped few variables that have some sort of xss potential

I was ambiguous, but was it rendered as HTML or plain text with HTML tags (like on Firefox)?

**Comment by amirfefer** [comment]
@ekohl  rendered as plain text (like firefox)

File: app/assets/javascripts/charts.js (hunk 1 above)
Comment (amirfefer) [reply app/assets/javascripts/charts.js]: I guess it would :+1:

**Review by ekohl — APPROVED** [review]

**Comment by waldenraines** [comment]
[test katello]

**Comment by mmoll** [comment]
merged, thanks @amirfefer!

**Pull request merged**